src/main/java/com/google/zxing/integration/android/IntentResult.java

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Please use Gradle dependency instead of pulling in unnecessary third-party code.

```
compile 'com.google.zxing:android-integration:3.2.1'
```

Also, have you considered [GPS barcode scanner](http://android-developers.blogspot.com.by/2015/08/barcode-detection-in-google-play.html)? Most likely it will remove dependency on third-party application for scanning.